sematic/api/endpoints/request_parameters.py

```diff
@@ -339,13 +343,61 @@ def _get_sql_predicates(
         operator = dict(AND=sqlalchemy.and_, OR=sqlalchemy.or_)[operand]
         return operator(
             *[
-                _extract_single_predicate(filter_, column_mapping)
+                _extract_predicate(filter_, column_mapping, model)
                 for filter_ in filters[operand]
             ]
         )
     else:
         filter_ = cast(ColumnPredicate, filters)
-        return sqlalchemy.and_(_extract_single_predicate(filter_, column_mapping))
+        return sqlalchemy.and_(_extract_predicate(filter_, column_mapping, model))
+
+
+def _extract_predicate(
+    filter: ColumnPredicate, column_mapping: ColumnMapping, model: type
+):
+    column_name = list(filter.keys())[0]
+
+    if "." in column_name:
```

Comment:
Please document this behavior in the function docstring.

Reply:
Thanks. Added such comments.